### DISCUSSION TITLE
Search within discussion

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#106769] [Mobile]
GitHub Mobile beta for Android

Load:
* https://github.com/community/community/discussions/3731

Tap ⋮

![Screenshot_20240208-085610.png](https://github.com/community/community/assets/2119212/a2cccb21-ff8e-4b3d-8c34-ee8b2d4de728)

There should be a find 🔍 option to search within the currently opened discussion.

Ideally it would have the ability to search collapsed content as well, although that could perhaps be optional especially as some content is collapsed because it's spam (note: sometimes I'm looking for the spam because I'm trying to report the spamming accounts).

There's a generic and unloved request that may be related, but it isn't deemed as answered, so I'm assuming this is worth reporting separately... 
* https://github.com/orgs/community/discussions/2828

**Comment by humixx** [comment]
I agree, there should be an option, Spam accounts ruin the discussion so bad:)
